src/main/java/com/redhat/cloud/policies/app/rest/AdminService.java

```diff
@@ -61,6 +62,9 @@ public class AdminService {
 
     private final Logger log = Logger.getLogger(this.getClass().getSimpleName());
 
+    @Inject
+    LightweightEngineConfig lightweightEngineConfig;
```

Comment:
ok. maybe that is one of the reasons why it is not in the engine but in a separate config class instead.

Reply:
One of the reasons explained above, yes 😃